src/components/search-filter/search-filter.stories.js

```diff
@@ -0,0 +1,111 @@
+import { html } from "lit-html";
+import "../button";
+import "../select";
+import "../input-field";
+import "../search-filter";
+import "../link";
+import styles from "./styles.scss";
+import buttonStyles from "../button/styles.scss";
+import inputStyles from "../input-field/styles.scss";
+import formStyles from "../form/styles.scss";
+import selectStyles from "../select/styles.scss";
+import linkStyles from "../link/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+
+export default {
+  title: "native-elements/vl-search-filter",
+  decorators: [
+    (story) =>
+      html`${stylesheet(
+        `${styles}${buttonStyles}${inputStyles}${formStyles}${selectStyles}${linkStyles}`
+      )}${story()}`,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["search-filter"],
+          root: "search-filter",
+          intro: "Search filter component.",
```

Comment:
Geen waanzinnig duidelijke intro, WU: `The search filter allows the user to refine the search results for a specific query.`

Reply:
Fixed.